docker/candlepin-base/cp-test

```diff
@@ -246,6 +254,14 @@ fi
 # Set our project to test
 PROJECT=${PROJECT:-server}
 
+# Update tomcat config to use Java 11 &
+# explicitly set Java 11 as default
+if [ "$JAVA_VERSION" == "11" ]; then
+    echo "Overriding tomcat.conf to use Java 11"
+    update-alternatives --set java /usr/lib/jvm/java-$JAVA_VERSION-openjdk-$JAVA_VERSION*/bin/java
+    sed -i 's/JAVA_HOME=.*/JAVA_HOME="\/usr\/lib\/jvm\/jre-11"/' /etc/tomcat/tomcat.conf
```

Comment:
Why are we doing this twice? This is already done above

Reply:
Logic changed. Not need anymore.